src/content_delegate.js

```diff
@@ -361,9 +361,20 @@ ContentDelegate.prototype.onDocumentViewSubmit = function (event) {
       // dcd (and presumably others) trigger this code path.
       var reader = new FileReader();
       reader.onload = function() {
+          let html=reader.result;
+          // check if we have an HTML page which redirects the user to the PDF
+          // this was first display by the Northern District of Georgia
+          // https://github.com/freelawproject/recap/issues/277
+          const redirectResult = Array.from(html.matchAll(/(?:window\.location\s=\s")(.*)(?:\")/g));
+          if(redirectResult && redirectResult.length>0) {
+            let url = redirectResult[0][1];
+            html = '<style>body { margin: 0; } iframe { border: none; }' +
+                 '</style><iframe src="' + url +
+                 '" width="100%" height="100%"></iframe>';
+          }
           this.showPdfPage(
-            document.documentElement, reader.result, previousPageHtml,
-            document_number, attachment_number, docket_number);
+              document.documentElement, html, previousPageHtml,
+              document_number, attachment_number, docket_number);
```

Comment:
Why is there a whitespace change here? And of course, under ordinary circumstances whitespace changes should be a separate commit, but maybe that's too picky here.

Reply:
A mistake on this one. I've removed the extra space.